homeassistant/components/sensor/device_automation.py

```diff
@@ -0,0 +1,217 @@
+"""Provides device automations for lights."""
+import logging
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library and 3rd party imports.

Reply:
Fixed.